meilisearch-core/src/bucket_sort.rs

```diff
@@ -559,15 +645,20 @@ impl<'txn> PostingsListView<'txn> {
 
     pub fn range(&self, range_offset: usize, range_len: usize) -> PostingsListView<'txn> {
         match self {
-            PostingsListView::Original { input, postings_list, offset, len } => {
+            PostingsListView::Original {
+                input,
+                postings_list,
+                offset,
+                len,
+            } => {
```

Comment:
I prefer the previous state :(

Reply:
Yeah it feels weird sometimes, but overall I hope it will force us to write nicer code.

Reply:
+ Ra hints the types now, it can be useful